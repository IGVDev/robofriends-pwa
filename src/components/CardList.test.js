import React from "react";
import CardList from "./CardList";
import { shallow, mount, render } from "enzyme";
import "../setupTests";

it("expect to render Card component", () => {
  const mockRobots = [
    {
      id: 1,
      name: "Jon Snow",
      username: "JonSnow",
      email: "jon@gmail.com",
    },
  ];
  expect(shallow(<CardList robots={mockRobots} />).debug()).toMatchSnapshot();
});
