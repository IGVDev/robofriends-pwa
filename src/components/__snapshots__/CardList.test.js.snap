// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Card component 1`] = `
"<div>
  <Card id={1} name=\\"Jon Snow\\" email=\\"jon@gmail.com\\" />
</div>"
`;
